lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3881,7 +3908,7 @@ void SwiftASTContext::ValidateSectionModules(
 
     // We have to increment the completion value even if we can't get the module
     // object to stay in-sync with the total progress reporting.
-    progress.Increment(++completion);
+    progress.Increment(++completion, llvm::formatv("{0}", module.GetFileSpec().GetFilename().AsCString()));
```

Comment:
No need for `llvm::formatv` here, you can just pass the call to `AsCString()`.

Reply:
I guess if you remove this, you can also drop the `include "llvm/Support/FormatVariadic.h"`

Reply:
Oh nice, I was using the formatters from how the progress reports were initially created so I'll change this.